src/OpenColorIO/Config.cpp

```diff
@@ -1094,414 +1100,6 @@ class Config::Impl
         // That should never happen.
         return -1;
     }
-
```

Comment:
Lines 1097-1504 were moved to ConfigUtils.cpp, without significant changes.  This is the heuristics code and we wanted to move this out of Config.cpp, since this module is already so long and the heuristics are essentially their own somewhat self-contained set of functions.